### DISCUSSION TITLE
navigatie

### DISCUSSION THREAD

**Discussion by Zuijdam** [geensnor/geensnor.nl-oud#18] [Ideas]
we moeten misschien de navigatie eens herzien. We kunnen niet meer zo goed uitbreiden en de verdeling tussen 'binnen de blog/jekyll' en daarbuiten (in de uitklap) is technisch wel kloppend maar voor de bezoeker wellicht minder evident.

Daarnaast misschien ook kijken naar andere benamingen ipv 'alle berichten'.....alleen weet ik niks beters :)

**Comment by reithose** [comment]
Goed om over na te denken inderdaad. 

Een idee:
- Wat nu in het hamburger menu staat, een aparte pagina van maken. Geeft ook de mogelijkheid om wat toelichting bij de andere links te geven.
- Gezichten in borden, klashoekje, tips, etc. naar het hamburger menu verplaatsen.
- In de hoofdnavigatie alleen "Home", "Alle berichten" en "Meer geensnor" (link naar de pagina van de bovenste bullet). Alle berichten zou ik wel in het hoofdmenu houden omdat dat belangrijk is om naar de andere berichten te komen of om te zoeken.

Op die manier komt alles een niveau dieper te liggen, maar ontstaat er wel meer ruimte in het hoofdmenu.

**Reply by Zuijdam** [reply]
ander idee is om het te doen net als gizmodo. Die hebben een soort overkoepelende navigatie over al hun brands heen zoals
https://gizmodo.com/
https://deadspin.com/

Die zouden we dan moeten implementeren in alle sites dan weer. Maar dat is sowieso misschien wel een aardig idee omdat je vanuit, neem een watetenwevandaag, niet zo handig naar geensnor komt.

**Reply by Zuijdam** [reply]
Er gebeuren trouwens wel rare dingen met die navigatie op gizmodo, dat moeten wij niet :) zie hoe het soms wel verspringt in volgorde en soms niet, afhankelijk waar je bent
